master/buildbot/reporters/gerrit_verify_status.py

```diff
@@ -135,6 +136,8 @@ def createStatus(self,
         if duration is not None:
             payload['duration'] = duration
 
+        payload = {'verifications': {name: payload}}
```

Comment:
No worries, if it did work then I'm willing to bet the API changed or something.

I'm a little curious how you did your end to end testing? I've got another project where I'm trying to get [gerrit setup with Docker](https://github.com/andygrunwald/go-gerrit/issues/11) and I ended up developing [gerrittest](https://github.com/opalmer/gerrittest) as a result. Might be useful for buildbot for these kinds of changes.